src/Apps/Order/Routes/Payment/orderPaymentReducer.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  OrderPaymentState,
+  OrderPaymentAction,
+  OrderPaymentActions,
+} from "./OrderPaymentContext"
+
+export const orderPaymentReducer = (
+  state: OrderPaymentState,
+  action: OrderPaymentAction
+) => {
+  switch (action.type) {
+    case OrderPaymentActions.SET_SELECTED_BANK_ACCOUNT_ID: {
+      return {
+        ...state,
+        selectedBankAccountId: action.payload,
+      }
+    }
+    case OrderPaymentActions.SET_SELECTED_PAYMENT_METHOD: {
```

Comment:
**suggestion:** Would it make sense to set `selectBankAccountId` to `null` if the selected payment method is credit card or wire transfer?

Reply:
that's a good suggestion. for now we just wanted to see how the whole flow would work, we haven't put much effort into the reducer logic yet. will do this for sure!